definitions/json/CaseEvent/CaseEvent-Rule92.json

```
@@ -73,5 +73,21 @@
     "CallBackURLAboutToStartEvent": "",
     "CallBackURLAboutToSubmitEvent": "",
     "CallBackURLSubmittedEvent": ""
+  },
+  {
+    "CaseTypeID": "ET_Scotland",
+    "ID": "STORE_RESPONDENT_TSE",
+    "Name": "Store respondent application",
+    "Description": "Store respondent application",
+    "PreConditionState(s)": "*",
+    "PostConditionState": "*",
+    "SecurityClassification": "Public",
+    "ShowEventNotes": "N",
+    "ShowSummary": "N",
+    "EventEnablingCondition": "caseType=\"dummy\"",
+    "CallBackURLAboutToStartEvent": "",
+    "CallBackURLAboutToSubmitEvent": "",
+    "CallBackURLSubmittedEvent": "",
+    "Publish": "Y"
```

No need to publish as not WA related
```suggestion
    "CallBackURLSubmittedEvent": ""
```